.github/workflows/ci-libshortfin.yml

```diff
@@ -116,13 +116,22 @@ jobs:
       if: "runner.os == 'Windows'"
       uses: ilammy/msvc-dev-cmd@0b201ec74fa43914dc39ae48a89fd1d8cb592756 # v1.13.0
 
-    - name: Setup Rust
-      # For now, `SHORTFIN_ENABLE_TOKENIZERS` is only enabled for 'Ubuntu (Clang)(full)'.
-      # TODO(#620): Enable on Windows.
-      if: ${{ matrix.name == 'Ubuntu (Clang)(full)'}}
-      uses: dtolnay/rust-toolchain@56f84321dbccf38fb67ce29ab63e4754056677e0 # master branch (Nov 18, 2024)
+    - name: Checkout IREE repo
+      uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
       with:
-        toolchain: stable
+        repository: iree-org/iree
+        path: ${{ env.IREE_REPO_DIR }}
+        submodules: false
+        ref: iree-3.1.0rc20241204
+
+    - name: Initalize IREE submodules
+      working-directory: ${{ env.IREE_REPO_DIR }}
+      run : |
+        git submodule update --init --depth 1 -- third_party/benchmark
+        git submodule update --init --depth 1 -- third_party/cpuinfo/
+        git submodule update --init --depth 1 -- third_party/flatcc
+        git submodule update --init --depth 1 -- third_party/googletest
+        git submodule update --init --depth 1 -- third_party/hip-build-deps/
```

Comment:
Looks like the revert was not clean. Only the "Setup Rust" step should be removed. These other checkout / submodule steps should not be added.

Reply:
this change should also remain

```diff
+              -DCMAKE_C_COMPILER=clang-18 -DCMAKE_CXX_COMPILER=clang++-18 -DCMAKE_LINKER_TYPE=LLD
-              -DCMAKE_C_COMPILER=clang-18 -DCMAKE_CXX_COMPILER=clang++-18 -DCMAKE_LINKER_TYPE=LLD -DSHORTFIN_ENABLE_TOKENIZERS=ON
```

Reply:
Technically only the CMake flag change is needed to disabling the native tokenizers component that is causing issues with CMake 4.0.0, but the native tokenizers component is the only thing using Rust, so we can also remove the "setup Rust" step.